src/components/LinearSchedule.vue

```diff
@@ -1,6 +1,6 @@
 <template lang="pug">
 .c-linear-schedule(v-scrollbar.y="")
-	.bucket(v-for="({date, sessions}, index) of sessionBuckets")
+	.bucket(v-if="sortBy == 'time'", v-for="({date, sessions}, index) of sessionBuckets")
```

Comment:
There is comment about using `v-for` with `:key` but you still don't update?

Reply:
Hi @hongquan, fixed as your suggestion, please help me review again